### PR TITLE
fix(grpc-transcode): ienum_as_name to enum_as_name

### PR DESCRIPTION
**Pull request by hnlq715**
### Pre-submission checklist:

* [] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: apisix/plugins/grpc-transcode.lua [hunk 1]
```diff
@@ -31,7 +31,7 @@ local pb_option_def = {
     },
     {   description = "int64 as result",
         type = "string",
-        enum = {"ienum_as_name", "enum_as_value"},
+        enum = {"enum_as_name", "enum_as_value"},
```
Comment (spacewander) [inline apisix/plugins/grpc-transcode.lua:34]: Can we add a test for it?

**Comment by moonming** [comment]
ping @hnlq715 @nic-chen

**Comment by nic-chen** [comment]
@hnlq715 

need test case for it, you could refer to :
https://github.com/apache/apisix/blob/master/t/plugin/grpc-transcode.t#L354-L413

if you need any help, please let us know, thanks.

**Comment by hnlq715** [comment]
@nic-chen 
Afaik, we need to add enum define in https://github.com/api7/grpc_server_example, so we can add test case for enum in https://github.com/apache/apisix/blob/master/t/plugin/grpc-transcode.t#L354-L413

**Comment by spacewander** [comment]
@hnlq715 
We can add a test just check the schema so there is no need to change the backend.

**Comment by hnlq715** [comment]
> @hnlq715
> We can add a test just check the schema so there is no need to change the backend.

OK, test case added.

**Review by spacewander — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**